src/main/java/mekanism/common/registries/MekanismModules.java

```diff
@@ -85,12 +86,12 @@ private MekanismModules() {
           ModuleGeigerUnit::new, () -> MekanismItems.MODULE_GEIGER.asItem(), builder -> builder.rarity(Rarity.UNCOMMON).rendersHUD());
     public static final ModuleRegistryObject<ModuleJetpackUnit> JETPACK_UNIT = MODULES.register("jetpack_unit",
           ModuleJetpackUnit::new, () -> MekanismItems.MODULE_JETPACK.asItem(), builder -> builder.rarity(Rarity.RARE).handlesModeChange().rendersHUD()
-                .exclusive(ExclusiveFlag.getFlags(ExclusiveFlag.OVERRIDE_JUMP)));
+                .exclusive(ExclusiveFlag.OVERRIDE_JUMP.getMask()));
```

Comment:
Thoughts on a helper in the builder to just take a vararg of exclusive flag?

Reply:
I like it, I'm thinking we just have it as a wrapper for the other call though. That also takes care of the deprecation I think.